submissions/akulahere/task-js-post-oop/index.js

```diff
@@ -0,0 +1,65 @@
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
```

Comment:
This makes all inhabitants have hands, not zero but no hands at all.
Imagine, we introduce birds. This way the code will become a nightmare soon, say apart constructors that need to keep `0` constants for hands, wings, horns etc

Reply:
refactored a little to avoid this problem